docs/src/modules/components/AppFrame.js

```diff
@@ -273,6 +288,53 @@ function AppFrame(props) {
                 anchorEl={languageMenu}
                 open={Boolean(languageMenu)}
                 onClose={handleLanguageMenuClose}
+                PaperProps={{
+                  variant: 'outlined',
+                  sx: {
+                    mt: 0.5,
+                    minWidth: 180,
+                    backgroundImage: 'none',
+                    borderColor: (theme) =>
+                      theme.palette.mode === 'dark' ? 'primaryDark.700' : 'grey.200',
+                    bgcolor: (theme) =>
+                      theme.palette.mode === 'dark' ? 'primaryDark.900' : 'background.paper',
+                    boxShadow: (theme) =>
+                      `0px 4px 20px ${
+                        theme.palette.mode === 'dark'
+                          ? 'rgba(0, 0, 0, 0.5)'
+                          : 'rgba(170, 180, 190, 0.3)'
+                      }`,
+                    '& .MuiMenuItem-root': {
+                      fontSize: (theme) => theme.typography.pxToRem(14),
+                      fontWeight: 500,
+                      '&:hover': {
+                        color: (theme) =>
+                          theme.palette.mode === 'dark' ? '#fff' : theme.palette.common.black,
+                        backgroundColor: (theme) =>
+                          theme.palette.mode === 'dark'
+                            ? alpha(theme.palette.primaryDark[700], 0.4)
+                            : theme.palette.grey[50],
+                      },
+                      '&:focus': {
+                        backgroundColor: (theme) =>
+                          theme.palette.mode === 'dark'
+                            ? alpha(theme.palette.primaryDark[700], 0.4)
+                            : theme.palette.grey[50],
+                      },
+                      '&.Mui-selected': {
+                        fontWeight: 500,
+                        color: (theme) =>
+                          theme.palette.mode === 'dark'
+                            ? theme.palette.primary[300]
+                            : theme.palette.primary[600],
+                        backgroundColor: (theme) =>
+                          theme.palette.mode === 'dark'
+                            ? theme.palette.primaryDark[700]
+                            : alpha(theme.palette.primary[100], 0.6),
```

Comment:
@siriwatknp We have an issue with how the customization of the color is working https://mui.com/customization/color/#playground. If you use it and open this menu (language menu) it crashes. There are also many styles that are wrong.

Reply:
Inconsistent in terms of visuals or code? Can you point out examples so we can open a new PR to fix it?

Reply:
If you could also explain to me what's the issue with this piece of code, I'd love to understand more!

Reply:
I was exploring this feedback https://twitter.com/MaterialUI/status/1473587799689465856?t=HWT4wGTUHZrZhJFYh2SXsQ

Reply:
> Inconsistent in terms of visuals or code? 

For instance:

<img width="390" alt="Screenshot 2021-12-23 at 15 28 00" src="https://user-images.githubusercontent.com/3165635/147253904-ce3de298-341d-4512-8733-c2d2fdccb964.png">

on https://mui.com/components/data-grid/

Actually, we have an issue open about it #30343.

Reply:
Okay, so it's about functionality 👍